### PR TITLE
Correct Softmax Error messages

### PR DESCRIPTION
**Pull request by joyalbin**

### CODE REVIEW

File: tensorflow/lite/kernels/activations.cc [hunk 1]
```diff
@@ -764,9 +764,10 @@ TfLiteStatus SoftmaxEval(TfLiteContext* context, TfLiteNode* node) {
         Softmax4DFloat(input, output, params);
         return kTfLiteOk;
       }
-      context->ReportError(
-          context, "Only 1D, 2D and 4D tensors supported currently, got %dD.",
-          NumDimensions(input));
+      context->ReportError(context,
+                           "Only 1D, 2D, 3D and 4D tensors supported currently "
+                           "Float32 kernels, got %dD.",
+                           NumDimensions(input));
```
Comment (lu-wang-g) [inline tensorflow/lite/kernels/activations.cc]: It will be more clear to change the wording to:
"The input tensor is 2D. Only 1D, 2D, 3D and 4D tensors are supported currently in Float32 kernel."

Same for the other logs.

Comment (joyalbin) [reply tensorflow/lite/kernels/activations.cc]: @lu-wang-g Input tensor can be any of dimension 1D, 2D, 3D and 4D right?
Should I add only "Only 1D, 2D, 3D and 4D tensors are supported currently in Float32 kernel, got %dD."

**Comment by tensorflowbutler** [comment]
Nagging Reviewers @lu-wang-g: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Review by lu-wang-g — APPROVED** [review]

**Comment by gbaned** [comment]
@joyalbin can you please check build failures

**Comment by gbaned** [comment]
@joyalbin Could you please check build failures and resolve the conflicts? Thanks!

**Comment by gbaned** [comment]
@joyalbin Gentle ping to check build failures and resolve the conflicts? Thanks!

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
